src/components/CallView/CallView.vue

```diff
@@ -370,6 +380,19 @@ export default {
 		supportedReactions() {
 			return getTalkConfig(this.token, 'call', 'supported-reactions')
 		},
+
+		/**
+		 * Fallback style for versions before v29.0.4
+		 */
+		callContainerStyle() {
+			if (serverSupportsBackgroundBlurred) {
+				return
+			}
+
+			return {
+				'backdrop-filter': this.isBackgroundBlurred ? 'blur(25px)' : 'none',
```

Comment:
`this.isBackgroundBlurred` is a ref object and you should test against `this.isBackgroundBlurred.value`

Reply:
.value is only valid for composition api code